docs/plugins/event-list.md

```diff
@@ -0,0 +1,31 @@
+---
+id: EventList
+title: List of Plugin Events
+sidebar_label: List of Plugin Events
+slug: /plugins/EventList
+---
+
+
+# List of Plugin Events
+
```

Comment:
Please add an explanation about the hierarchy of the list and the fact that there is a meaning to the order.

Reply:
Something like:
The following list represents the plugin events hierarchy i.e. the execution order of the events on our system